wiki/pages/Building-Packages.md

```diff
@@ -88,6 +88,11 @@ For example, our *boost* package's commands might look like so:
             # there is a 'FindBoost.cmake' file in this dir..
             env.CMAKE_MODULE_PATH.append("{root}/cmake")
 
+When *boost* is required into another package's build environment, `building` will be set to
+*True*, thereby executing that block of code, but otherwise, it will not be executed, even when
```

Comment:
I'm finding this a bit confusing because it's conflating two different things - what 'building' is for, and the fact that a package's commands don't get executed if it's the package itself getting built. But, this is still a good place to highlight that info.

Perhaps a separate warning would be good:

```
> [[media/icons/warning.png]] Note that _commands_ is never executed for the package actually getting built - 
> if you want to run commands in that case, you use [pre_build_commands](Package-Commands#pre-build-commands) instead.
```